src/test/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwingTest.java

```diff
@@ -0,0 +1,58 @@
+package com.acrolinx.sidebar.swing;
+
+import com.acrolinx.sidebar.AcrolinxIntegration;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.awt.event.KeyEvent;
+
+class AcrolinxSidebarSwingTest {
+
+    @Test
+    void processKeyEvent_PasteOnMacIsConsumed() {
```

Comment:
we should not use "_" in method names, SonarLint also raises warnings about this

Reply:
I thought this allowed for test classes, it improves readability I'm my opinion.

Reply:
we should use the same naming conventions/standards for main and test code. I think SonarLint uses ^[a-z][a-zA-Z0-9]*$ to check method names (except for overridden ones, where we cannot change them).